### PR TITLE
Create about-nodes.md

### PR DESCRIPTION
**Pull request by Phyloiota**
# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancment

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: docs/participate/support-the-network/about-nodes.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## Synchronizing with the rest of the network
+
+Like any distributed system, nodes in an IOTA network synchronize their databases with others called neighbors to form a single source of truth.
+
+When one node, no matter where it is in the world, receives a transaction, it will try to gossip it to all its neighbors. This way, all nodes eventually see all transactions and store them in their local databases.
+
+To synchronize, nodes in IOTA networks use milestones.
+If the node has the history of transactions that a milestone references, that milestone is solid.
+
+Therefore, nodes know if they are synchonized if the index of their latest solid milestone is the same as the index of the latest milestone that it has received.
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/about-nodes.md:36]:
```suggestion
Therefore, nodes know if they are synchronized if the index of their latest solid milestone is the same as the index of the latest milestone that it has received.
```

File: docs/participate/support-the-network/about-nodes.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+When a node is synchronized, it then has enough information to decide which transactions it considers confirmed.
+
+## Deciding which transactions are confirmed
+
+All transactions remain in a pending state until the node is sure of its validity. For a definition of a valid transaction, see **Value Transfer**.
+
+However, even when a transaction is valid, nodes may not be able to make a decision like in the case of a double spend.
+
+When nodes detect double spends they must decide which transaction to consider confirmed and which one to ignore. Nodes do this by using consensus rules that are built into their node software. See **The Coordinator** for information about the consensus rules in IOTA networks.
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/about-nodes.md:46]: Should "The Coordinator" be a link?

Comment (Dr-Electron) [inline docs/participate/support-the-network/about-nodes.md:46]:
```suggestion
When nodes detect double spends, they must decide which transaction to consider confirmed and which one to ignore. Nodes do this by using consensus rules that are built into their node software. See **The Coordinator** for information about the consensus rules in IOTA networks.
```

File: docs/participate/support-the-network/about-nodes.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Keeping a record of the balances on addresses
+
+All nodes keep a record of the balances of addresses so they can do the following:
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/about-nodes.md:50]:
```suggestion
All nodes keep a record of the balances of addresses, so they can do the following:
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: docs/participate/support-the-network/about-nodes.md (hunk 2 above)
Comment (Phyloiota) [reply docs/participate/support-the-network/about-nodes.md]: yes - ill add the link when we have a page for that (or another content that describes the coordinator)

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**